css/BEM-style.md

```diff
@@ -19,9 +19,9 @@ About 30 - 45 minutes
 
 On smaller pet projects, the way you organise your CSS or even SaSS styles doesn't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time.
 
-However in bigger projects with over a 100 stylesheets and varied components spread across the application, that can raise a serious issue, since changing the styles for any compponent would mean searching for the absurd CSS naming (either id's, classes or combination of both ) you provided maybe a month before.
+However in bigger projects with over a 100 stylesheets and varied components spread across the application, that can raise a serious issue, since changing the styles for any component would mean searching for the absurd CSS naming (either id's, classes or combination of both) you provided maybe a month before.
```

Comment:
```suggestion
However in bigger projects with over a 100 style sheets and varied components spread across the application, that can raise a serious issue, since changing the styles for any component would mean searching for the absurd CSS naming (either id's, classes or combination of both) you provided maybe a month before.
```